### PR TITLE
add aionotify recipe

### PR DESCRIPTION
**Pull request by hodgsonw**

### CODE REVIEW

**Comment by epruesse** [comment]
LGTM
ping @conda-forge/help-python

File: recipes/aionotify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - pip
+    - python >=3.4
+  run:
+    - python >=3.4
+
+test:
+  imports:
+    - aionotify
+  requires:
+    - asynctest
```
Comment (synapticarbors) [inline recipes/aionotify/meta.yaml]: Since you're not running the actual test suite, `asynctest` shouldn't be needed

Comment (hodgsonw) [reply recipes/aionotify/meta.yaml]: the changes have been made

**Comment by epruesse** [comment]
ping @conda-forge/help-python

**Pull request merged**